package.json

```diff
@@ -115,12 +115,12 @@
   },
   "homepage": "https://renovatebot.com",
   "engines": {
-    "node": "^12.22.0 || >=14.15.0",
+    "node": "^14.15.0 || >=16.13.0",
```

Comment:
https://github.com/renovatebot/renovate/blob/665b3d359cf5db486550afb32d593de2096386b9/docs/usage/getting-started/running.md?plain=1#L30

If you're going to disallow node `v17` maybe that should also be listed in the docs?

Reply:
It's  only a warning, not fully disallowed 😉  I think renovate currently won't fully work on node v17.

Reply:
I've been debugging okay on node 17 🤷

Reply:
So should we don't warn on node v17?

Reply:
I don't think so. What are the issues you were expecting to encounter? I wasn't looking for anything.

Reply:
I think we officially only support node lts versions 🤷‍♂️ , so thats why is like to warn about non lts 🙃

Reply:
@rarkins WDYT?

Also review the changes! `engines` allows node v17. We only issue a log warning about node v17 here:

https://github.com/renovatebot/renovate/blob/b350a8c5fb660f036d06ec85a4193fca8585f4a3/lib/workers/global/index.ts#L55-L74

Reply:
I thought we supported current (v17), active LTS (v16), maintenance LTS (v14)? I haven't seen that message because v17 satisfies the `>=14.15.0` part of that range.

Reply:
I don't think we can support to current because sometimes it's "broken" for us?

I'm OK with allowing it to run in current but having a policy that we won't debug/troubleshoot it.

Reply:
There's kind of three thresholds we can have:

1. `engines`: will block from running
2. `engines-next`: will log a warning
3. Our documentation: just merely a clarification on our project policy that we don't debug or troubleshoot non-LTS versions